scripts/source-maintenance.sh

```diff
@@ -62,8 +69,8 @@ else
 fi
 
 ASVER=`astyle --version 2>&1 | grep -o -E "[0-9.]+"`
-if test "${ASVER}" != "2.04" ; then
-	echo "Astyle version problem. You have ${ASVER} instead of 2.04"
+if test "${ASVER}" != "${TargetAstyleVersion}" ; then
+	echo "Astyle version problem. You have ${ASVER} instead of ${TargetAstyleVersion}"
```

Comment:
Something went wrong in this change. The original warning (the one specific to v2.04) should be preserved (and hopefully reworded to be more informative). If that warning is no longer needed when a user copy-pastes a source-maintenance command from somewhere, then the "version insistence" feature itself is no longer needed -- we can just format with whatever astyle we happen to find!